src/mixins/org-person-mixin.ts

```diff
@@ -84,6 +84,11 @@ export default class OrgPersonMixin extends Vue {
     return (person.actions?.includes(ActionTypes.REMOVED))
   }
 
+  /** Returns True if the specified orgPerson was replaced. */
+  wasReplaced (person: OrgPersonIF): boolean {
+    return (person.actions?.includes(ActionTypes.REPLACED))
+  }
```

Comment:
Note that a replaced org-person also has the ADDED or REMOVED action, so you can tell whether it's the new org-person or the old one.